packer/vhd-image-builder.json

```diff
@@ -191,6 +191,11 @@
             "source": "parts/k8s/cloud-init/artifacts/docker_clear_mount_propagation_flags.conf",
             "destination": "/home/packer/docker_clear_mount_propagation_flags.conf"
         },
+        {
+            "type": "file",
+            "source": "releases/notice/notice.txt",
+            "destination": "/NOTICE.txt"
```

Comment:
Do we really want to put this in the root volume?

Reply:
That seems to be the case, I can check if alternative locations are accepted.

Reply:
BTW, pushed a commit to validate if creating the file in the root vol is causing the pipeline to fail